openerp/addons/base/i18n/base.pot

```diff
@@ -10041,6 +10299,18 @@ msgid "Missing SMTP Server"
 msgstr ""
 
 #. module: base
+#: code:addons/models.py:6036
+#, python-format
+msgid "Missing required value for the field '%s' (%s)"
```

Comment:
I mean, as it new strings to translate (not all due to the move from `osv/osv.py` to `models.py`), better have them correct from the start.

I don't know if transifex recognise it but we can also use [`format` format](https://docs.python.org/2/library/string.html#formatstrings)